wallet-new/src/Cardano/Wallet/Kernel.hs

```diff
@@ -245,25 +245,29 @@ bracketActiveWallet walletProtocolMagic walletPassive walletDiffusion runActiveW
 --
 -- If the pending transaction is successfully added to the wallet state, the
 -- submission layer is notified accordingly.
-newPending :: ActiveWallet -> HdAccountId -> TxAux -> IO (Either NewPendingError ())
-newPending ActiveWallet{..} accountId tx = do
+newPending :: ActiveWallet -> HdAccountId -> TxAux -> Maybe TxMeta ->  IO (Either NewPendingError ())
```

Comment:
This still takes a `Maybe`. Why is that?

Reply:
@edsko only for tests. Some tests use it that are not ready for this